panacea/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.4'
+
+# Required configuration can be found on
+# https://medibloc.gitbook.io/panacea-core/guide/join-the-network 
+
+services:
+  node_1:
+    build:
+      context: ../
+      args:
+        PROJECT: panacea
+        PROJECT_BIN: panacead
+        PROJECT_DIR: .panacea
+        VERSION: v2.0.2
+        REPOSITORY: https://github.com/medibloc/panacea-core
+    ports:
+      - '1317:1317'
+      - '26656:26656'
+      - '26657:26657'
+    environment:
+      - MONIKER=node_1
+      - CHAIN_JSON=https://raw.githubusercontent.com/cosmos/chain-registry/master/panacea/chain.json
+      - GENESIS_URL=https://github.com/medibloc/panacea-mainnet/raw/master/panacea-3/genesis.json.gz?raw=true
```

Comment:
Can we remove GENESIS_URL and P2P_PERSISTENT_PEERS here? I assume they are the same as the values in the CHAIN_JSON so would be sourced from there

Reply:
I removed them, because indeed they are specified in the chain.json file. Tested and appears to be working without them too.